lib/hammer_cli_katello/filter.rb

```diff
@@ -53,6 +53,13 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       success_message _("Filter created")
       failure_message _("Could not create the filter")
 
+      validate_options do
+        organization_options = [:option_organization_id, :option_organization_name, \
+                                :option_organization_label]
+
+        any(*organization_options).required
+      end
```

Comment:
Organization currently is not required in order to try and create a content-view filter, if you give it all IDs, it will currently require it if you try and give it the name of a content-view. 

However recently the API was changed to not require Organization-id when trying to look up the id of a repository. So if a person were to give a repository-name without an organization-id it would fail. 

I don't think that this causes any harm, but thinking about it now I could change it to only require organization_options if either content-view name or repository name is provided.